hooks/useTBAPurchase.tsx

```diff
@@ -16,6 +17,18 @@ const useTBAPurchase = () => {
   const [loading, setLoading] = useState(false)
   const { isLoggedByEmail } = useUserProvider()
   const { prepare } = usePreparePrivyWallet()
+  const [preparedMulticalls, setPreparedMulticalls] = useState(null)
+
+  useEffect(() => {
+    const init = async () => {
+      const prepared = await getPreparedMulticalls(
+        (connectedWallet as `0x${string}`) || zeroAddress,
```

Comment:
can you use `Address` type from viem as well?
```suggestion
        (connectedWallet as Address) || zeroAddress,
```

Reply:
fixed ✅